integrations/make.rst

```diff
@@ -0,0 +1,135 @@
+.. _make:
+
+Make
+====
+
+Conan provides integration with plain Makefiles by means of the ``make`` generator. If you are using ``Makefile`` to build your project you
+could get the information of the dependencies in a *conanbuildinfo.mak* file. All you have to do is indicate the generator like this:
+
+.. code-block:: text
+   :caption: *conanfile.txt*
+
+    [generators]
+    make
+
+.. code-block:: text
+   :caption: *conanfile.py*
+
+    class MyConan(ConanFile):
+        ...
+        generators = "make"
+
+Example
+-------
+
+We are going to use the same example from :ref:`getting_started`, a MD5 Encrypter app.
+
+This is the main file for it:
+
+.. code-block:: cpp
+   :caption: main.cpp
+
+    #include "Poco/MD5Engine.h"
+    #include "Poco/DigestStream.h"
+
+    #include <iostream>
+
+
+    int main(int argc, char** argv)
+    {
+        Poco::MD5Engine md5;
+        Poco::DigestOutputStream ds(md5);
+        ds << "abcdefghijklmnopqrstuvwxyz";
+        ds.close();
+        std::cout << Poco::DigestEngine::digestToHex(md5.digest()) << std::endl;
+        return 0;
+    }
+
+As this project relies on the Poco Libraries we are going to create a *conanfile.txt* with our requirement and declare there the Make
+generator too:
+
+.. code-block:: text
+   :caption: conanfile.txt
+
+    [requires]
+    Poco/1.9.0@pocoproject/stable
+
+    [generators]
+    make
+
+In order to use this generator within your project, use the following Makefile as a reference:
+
+.. code-block:: makefile
+   :caption: Makefile
+
+    include conanbuildinfo.mak
+
+    #----------------------------------------
+    #     Make variables for a sample App
+    #----------------------------------------
+
+    CXX_SRCS = \
+    main.cpp
+
+    CXX_OBJ_FILES = \
+    main.o
+
+    EXE_FILENAME = \
+    main
+
+
+    #----------------------------------------
+    #     Prepare flags from variables
+    #----------------------------------------
+
+    CPPFLAGS        += $(addprefix -I, $(CONAN_INCLUDE_PATHS))
+    CPPFLAGS        += $(addprefix -D, $(CONAN_DEFINES))
+    LDFLAGS         += $(addprefix -L, $(CONAN_LIB_PATHS))
+    LIBS            += $(addprefix -l, $(CONAN_LIBS))
+
+
+    #----------------------------------------
+    #     Make Commands
+    #----------------------------------------
+
+    COMPILE_CXX_COMMAND         ?= \
+        g++ -c $(CXXFLAGS) $(CPPFLAGS) $< -o $@
```

Comment:
`$(CXXFLAGS) ` should just be `$(CFLAGS) ` since there is actually no variable called `$(CXXFLAGS)` in this example, nor is one propagated by Conan.  Conan only propagates a single variable called `CFLAGS`.  Technically, it doesn't matter here because there were no `CFLAGS` defined in this example, so both will result in an empty string, but we should definitely make the example correct.